glidertools/_version.py

```diff
@@ -0,0 +1 @@
+__version__ = "2021.4.dev25+g01092e3"
```

Comment:
This file should not be committed. I will add it to the .gitignore, but can you remove it here?

Reply:
Actually I am not sure If I should add it there...but can you still remove it?